pkg/restapi/vc/operation/models.go

```diff
@@ -61,6 +62,22 @@ type IssueCredentialOptions struct {
 	AssertionMethod string `json:"assertionMethod,omitempty"`
 }
 
+// ComposeCredentialRequest for composing and issuing credential.
+type ComposeCredentialRequest struct {
+	Issuer                  string                 `json:"issuer,omitempty"`
+	Subject                 string                 `json:"subject,omitempty"`
+	Types                   []string               `json:"types,omitempty"`
+	IssuanceDate            *time.Time             `json:"issuanceDate,omitempty"`
+	ExpirationDate          *time.Time             `json:"expirationDate,omitempty"`
+	Claims                  map[string]interface{} `json:"claims,omitempty"`
+	Evidence                verifiable.Evidence    `json:"evidence,omitempty"`
+	TermsOfUse              json.RawMessage        `json:"termsOfUse,omitempty"`
+	CredentialFormat        string                 `json:"credentialFormat,omitempty"`
+	ProofFormat             string                 `json:"proofFormat,omitempty"`
+	CredentialFormatOptions json.RawMessage        `json:"credentialFormatOptions,omitempty"`
+	ProofFormatOptions      json.RawMessage        `json:"proofFormatOptions,omitempty"`
```

Comment:
q: why are `ProofFormatOptions` and `CredentialFormatOptions` of type `json.RawMessage`, but `Claims` is a `map[string]interface{}`?

Reply:
Updated.